vsts/scripts/pushBaseImagesToRegistry.sh

```diff
@@ -32,3 +32,11 @@ if [ -f "$stretchBaseImagesFile" ]; then
     done <"$stretchBaseImagesFile"
 fi
 
+if [ -f "$bullseyeBaseImagesFile" ]; then
+    echo "$bullseyeBaseImagesFile exists. pushing bullseye image tags ..."
+    while read imageName; do
+      # read the tags from file
+      echo "Pushing image $imageName ..."
+      docker push "$imageName"
+    done <"$bullseyeBaseImagesFile"
+fi
```

Comment:
Do the tests that we currently have test that the platforms build and run successfully with the new Bullseye image?

Reply:
There are existing runtime tests that expect specific versions of the platform runtimes to be built and run tests against them -- this includes the two versions that have been migrated in this PR. However, I'm not sure if there's a way to test what Debian flavor we're using within the runtime image, so maybe that's something we can investigate.